sdcm/cluster.py

```diff
@@ -3138,16 +3143,16 @@ def node_setup(node):
             if self._experimental():
                 self._node_setup(node=node)
             node.wait_db_up(verbose=verbose)
-            node.remoter.run('sudo yum install -y scylla-gdb',
-                             verbose=verbose, ignore_status=True)
+            # node.remoter.run('sudo yum install -y scylla-gdb',
+            #                  verbose=verbose, ignore_status=True)
             queue.put(node)
             queue.task_done()
 
         start_time = time.time()
 
         for node in node_list:
             node.wait_ssh_up(verbose=verbose)
-            self.collectd_setup.install(node)
+            # self.collectd_setup.install(node)
```

Comment:
Same as above, unless we don't need the collectd anymore... 
@amoskong, do you know if we still need it?